pkg/test/framework/components/echo/deployment/builder.go

```diff
@@ -302,6 +303,10 @@ func (b *builder) getOrCreateNamespace(prefix string) (*builder, namespace.Insta
 func (b *builder) deployServices() (err error) {
 	services := make(map[string]string)
 	for _, cfg := range b.configs {
+		if b.ctx.Settings().EnableDualStack && cfg.IPFamilyPolicy == "" {
+			cfg.IPFamilies = "IPv4, IPv6"
```

Comment:
Since a dualStack K8s cluster can also be configured with `IPv6` as the default address family, do you think we should avoid specifying the IPFamilies here?

Reply:
I was planning to make ipv6 first, but then there are failures in ambient when I do that.. that is why I have tagged John and Ben to help analyse the failure..

Reply:
Yeah, saw that. I agree that its good to identify the reasons for failure and fix them, but my question is more about should we avoid specifying `IPFamilies` itself as a user can execute these tests on dualStack clusters where we can have either of the families configured as default address family.

Reply:
I also agree that it would be better not to override the svc with a specific default ip family in mind.
I feel that having the default family of the svc being the same as the default family of the cluster makes more sense and it would lead for better coverage in DS IPv6-default clusters where issues would otherwise go undetected.

Reply:
I think we have only ipv4 first dualstack cluster in the CI.. I wanted to try both the orders to see if all the tests pass . And already saw some failure when specifying ipv6 first

Reply:
Really, I think we should change how our test flags work.

Right now we have `IP_FAMILIES` which can be 

- "V4"
- "V6"
- "dual"

I suggest we simply pass a list of families to the test as a test arg and drop "dual".  Whatever `ipfamily` order the tests are passed is the order that will be used in the test, and if more than one family is provided we always `Require` rather than `Prefer` for the policy.

This simplifies things, as the only test knob is `IP_FAMILIES` which can be `V4`, or `V6`, or both, in your choice of order/preference.

Reply:
Sounds fine to me. Are you also proposing that we have similar support (i.e., [v4, v6] or [v6, v4]) even while deploying KIND clusters?